### PR TITLE
Use english language and force tests to be run in production mode

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Use english language and force tests to be run in production mode
| Type?         | improvement
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Travis must be green

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/14096)

### CODE REVIEW

File: tests/E2E/docker-compose.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ services:
       DB_SERVER: db
       PS_INSTALL_AUTO: 1
       PS_COUNTRY: fr
+      PS_LANGUAGE: en
```
Comment (Quetzacoalt91) [inline tests/E2E/docker-compose.yml:27]: Does this actually fix the issue? This value is already set by default: https://github.com/PrestaShop/docker/blob/c93fb39ae49bc8b4720577c0a6df8631e12eecfb/base/images/7.2-apache/Dockerfile#L13

Comment (PierreRambaud) [reply tests/E2E/docker-compose.yml:27]: I had the same reflexion, we add this (with @boubkerbribri ) to try if it's ok during the nightly

Comment (boubkerbribri) [reply tests/E2E/docker-compose.yml:27]: @PierreRambaud @Quetzacoalt91 I'm not sure about that.
You can have a look at the nightly report test full/08_stocks/4_update_product_quantity.js
We have this error 
``` AssertionError: expected 'Please enter a valid value. The two nearest valid values are 10 and 11.' to include 'Veuillez saisir une valeur valide.' ```
at /prestashop-core-nightly/reports/2019-06-05-1.7.6.x.html#040f4062-063a-4d51-8514-24e79b41efeb

Comment (PierreRambaud) [reply tests/E2E/docker-compose.yml:27]: We'll investigate to understand clearly what's happening :/

Comment (Quetzacoalt91) [reply tests/E2E/docker-compose.yml:27]: I'm not saying the shop is properly in English, I'm saying this change won't fix your issue. :)

Comment (boubkerbribri) [reply tests/E2E/docker-compose.yml:27]: > I'm not saying the shop is properly in English, I'm saying this change won't fix your issue. :)

Sorry for misunderstanding.

**Pull request closed without merging**